src/Response/XmlApiResponse.php

```diff
@@ -48,7 +55,7 @@ public function getCode()
     public function send(IRequest $httpRequest, IResponse $httpResponse)
     {
         $httpResponse->setContentType('text/xml');
-        $httpResponse->setExpiration(false);
+        $httpResponse->setExpiration($this->expiration);
```

Comment:
could this be null or have to be false in this version?

Reply:
if false, than use $this->expiration ?: false

Reply:
so this will not have a null value